src/diffusers/pipelines/audioldm/__init__.py

```diff
@@ -0,0 +1 @@
+from .pipeline_audioldm import AudioLDMPipeline
```

Comment:
re: https://github.com/huggingface/diffusers/pull/2232#discussion_r1117472834

This is where we should gate on transformers, torch, and transformers version

Reply:
Resolved in https://github.com/huggingface/diffusers/pull/2232/commits/9f776a2845abd457a6e471e1ef4aacb4c9005ba9